src/coreclr/gc/gcee.cpp

```diff
@@ -165,6 +177,52 @@ void GCHeap::UpdatePostGCCounters()
 
     FIRE_EVENT(GCEnd_V1, static_cast<uint32_t>(pSettings->gc_index), condemned_gen);
 
+    // Fire custom event for process private memory at GC end
+    uint64_t processPrivateMemory = 0;
+
+#ifdef TARGET_WINDOWS
```

Comment:
Should this be abstracted via GC PAL? e.g. https://github.com/dotnet/runtime/blob/main/src/coreclr/gc/unix/gcenv.unix.cpp . It is how most of the GC codebase deals with dependencies.